redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -68,16 +68,31 @@ import platform.darwin.NSObject
 internal open class UIViewLazyList :
```

Comment:
There is a lot of Swift and Kotlin stuff happening in this file. Rather than being precise and eliminating just the references from Swift objects to Kotlin ones, I just cut down the reference graph aggressively.

There’s still a few places where Swift objects will hold Kotlin objects...

The cells in the table will be instances of `LazyListContainerCell`. This is a UIView subclass implemented in Kotlin, and honestly I don’t have intuition on how reachability works with such things. Each cell has a `Binding` that I might also need to clear.

The table has a datasource and a delegate, both of which are weakly held. I also don’t have much intuition on how a Swift weak references to a Kotlin object is implemented.